packages/devextreme/js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -183,6 +183,7 @@ const TextEditorMask = TextEditorBase.inherit({
         this.$element().removeClass(TEXTEDITOR_MASKED_CLASS);
         this._maskRulesChain = null;
 
+        this._maskStrategy.clean();
```

Comment:
`clean` is a kind of opposite action for `init`. In this method I see that we added `clean` but we don't have `init` here. It means afaid that cleaned dragtimer, caretTimeout and inputHandlerTimeout is not recreated.

So I don't get why do we need this change and is it safe.

Reply:
Without this change, we get an error about uncleaned timeout.